client/src/frontend/components/composed/EntityHero/parts/CalloutList/Callout/params.js

```diff
@@ -6,7 +6,21 @@ export const getCalloutParams = (data, type, isLink) => {
   const tocSectionId =
     data.relationships?.text?.attributes.tocSectionId ?? null;
 
-  switch (type) {
+  const finalType = (() => {
+    if (type !== "LINK") return type;
+    switch (data.attributes.title) {
+      case "Website":
+        return "WEBSITE";
+      case "EMAIL":
+        return "EMAIL";
+      case "SHARE":
+        return "SHARE";
+      default:
+        return "LINK";
+    }
+  })();
```

Comment:
Not sure what the best solution is here. All of these are `type: LINK`, but they have unique icons in design. These strings certainly aren't going to catch all cases.

Reply:
Don't include icons for now. Have them behave as they do on existing project pages.